language_service/src/completion/tests.rs

```diff
@@ -174,84 +174,6 @@ fn ignore_unstable_callable() {
     );
 }
```

Comment:
Why is the only test change _removing_ tests. Isn't there any change in behavior to verify here?

Reply:
Added tests in b0f7de3